pkgs/apps/psi4/default.nix

```diff
@@ -211,7 +207,7 @@ buildPythonPackage rec {
     "-DCMAKE_INSIST_FIND_PACKAGE_gau2grid=ON"
     "-Dgau2grid_DIR=${gau2grid}/share/cmake/gau2grid"
     # libint
-    "-DMAX_AM_ERI=6"
+    "-DMAX_AM_ERI=4"
```

Comment:
Any special reason reduce the MAX_AM?

Reply:
Ahh this was a leftover from playing around to get build time down from 2 hours :grimacing: Back at 6.

Reply:
On the big AMD machine with 64 threads it takes about an hour.